server/express-router.md

```diff
@@ -0,0 +1,96 @@
+Previously, we've learnt to separate routes into their own file, by exporting them as functions and requiring them within our `server.js` file:
+
+```js
+/*
+ * server.js
+ */
+
+var express = require('express')
+var routes = require('./routes')
+
+var PORT = 3000
+var app = express()
+
+app.get('/', routes.index)
+
+app.listen(PORT, function () {
+  console.log('The server is listening on port', PORT)
+})
+
+/*
+ * routes.js
+ */
+
+module.exports = {
+  index: index
+}
+
+function index (req, res) {
+  res.send('<h1>Hello world</h1>')
+}
+```
+
+However, Express has built in functionality to take care of this also:
```

Comment:
reworded given the change made due to the previous comment:

"However, as we add more routes to our Express server, this can become cumbersome. Express has built in functionality to separate our routes and make our code more modular:"